layouts/partials/docs/breadcrumb.html

```diff
@@ -14,6 +14,8 @@
     {{ end }}
     <!-- If the page is a redirect page, don't include it in the breadcrumb. -->
     {{ if not (isset .p1.Params "redirect_to") }}
-        <li {{ if eq .p1 .p2 }}class="active"{{ end }}><a href="{{ .p1.Permalink }}">{{ .p1.LinkTitle }}</a></li>
+        <li {{ if eq .p1 .p2 }}class="active"{{ end }}>
+            <a data-track="breadcrumb" href="{{ .p1.Permalink }}">{{ .p1.LinkTitle }}</a>
+        </li>
```

Comment:
It looks like this change is causing a space to show up in front of bread crumb items for some reason. Not sure why the added whitespace here is causing that to happen.

http://content-bucket-a1a482d.s3-website-us-east-1.amazonaws.com/docs/intro/concepts/

<img width="410" alt="Screen Shot 2020-01-09 at 2 38 10 PM" src="https://user-images.githubusercontent.com/710598/72099142-fc509980-32ed-11ea-8d98-cd53d9a69ec2.png">

https://www.pulumi.com/docs/intro/concepts/

<img width="400" alt="Screen Shot 2020-01-09 at 2 40 59 PM" src="https://user-images.githubusercontent.com/710598/72099178-0ffc0000-32ee-11ea-88aa-6b2fa87c210d.png">

Reply:
I'm seeing this locally also.

Reply:
Moving the html back to one line.